esmvalcore/preprocessor/_derive/rsus.py

```diff
@@ -0,0 +1,41 @@
+"""Derivation of variable `rsus`.
+
+authors:
+    - lukas_brunner
+
+"""
+from iris import Constraint
+
+from ._baseclass import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `rsus`."""
+
+    @staticmethod
+    def required(project):
+        """Declare the variables needed for derivation."""
+        required = [
+            {
+                'short_name': 'rsds'
+            },
+            {
+                'short_name': 'rsns'
+            },
+        ]
+        return required
+
+    @staticmethod
+    def calculate(cubes):
+        """Compute upwelling shortwave flux from downwelling and net."""
+        rsds_cube = cubes.extract_cube(
+            Constraint(name='surface_downwelling_shortwave_flux_in_air'))
+        rsns_cube = cubes.extract_cube(
+            Constraint(name='surface_net_downward_shortwave_flux'))
+        # fix latitude and longitude var_name
+        rsns_cube.coord("longitude").var_name = "Longitude"
```

Comment:
Rather than making a choice, it would  probably be safer to just make sure they are the same for both variables, e.g.:
```suggestion
        rsns_cube.coord(axis='X').var_name = rsds_cube.coord(axis='X').var_name
```

Reply:
yeh that's not a bad idea at all - more general, and if the X-azis is called `cows` that'll get picked up by the checker (or not, since it's var_name anyway) :+1: